frontend/src/ui/components/ErrorPage/ErrorPage.tsx

```diff
@@ -4,15 +4,38 @@ import { WrappedMessage } from 'utils/intl';
 import messages from './displayMessages';
 import './styles.scss';
 
-export const ErrorPage: React.FC = () => (
-  <div className="error-page">
-    <div className="error-container">
-      <Row className="error-page-content">
-        <p>
-          <WrappedMessage id="somethingWrong" messages={messages} />
-        </p>
-        <WrappedMessage id="errorMessage" messages={messages} />
-      </Row>
+interface ErrorPageProps {
+  messages?: any;
+  messageId?: string;
+  values?: any;
+}
+
+export const ErrorPage: React.FC<ErrorPageProps> = (props: ErrorPageProps) => {
+  if (props.messages && props.messageId) {
+    return (
+      <div className="error-page">
+        <div className="error-container">
+          <Row className="error-page-content">
+            <WrappedMessage
+              id={props.messageId}
+              messages={props.messages}
+              values={props.values}
+            />
+          </Row>
+        </div>
+      </div>
+    );
+  }
+  return (
+    <div className="error-page">
+      <div className="error-container">
+        <Row className="error-page-content">
+          <p>
+            <WrappedMessage id="somethingWrong" messages={messages} />
+          </p>
+          <WrappedMessage id="errorMessage" messages={messages} />
+        </Row>
+      </div>
     </div>
```

Comment:
Nit: since most of the returned HTML structure is the same, can we pull the inner bits that are different into variables so that we can do something like:

```javascript
let body;
if (props.messsages && props.messageId) {
  body = <WrappedMessage id={...} ... />;
} else {
  body = <>
    <p>
      ...
    </p>
    <WrappedMessage ... />
  </>;
}

return (
  <div className="error-page">
    <div className="error-container">
      <Row className="error-page-content">
        {body}
      </Row>
    </div>
  </div>
);
```